contracts/account/utils/draft-ERC4337Utils.sol

```diff
@@ -8,6 +8,11 @@ import {Math} from "../../utils/math/Math.sol";
 import {Calldata} from "../../utils/Calldata.sol";
 import {Packing} from "../../utils/Packing.sol";
 
+/// @dev This is available on all entrypoint since v0.4.0, but is not formally part of the ERC.
+interface IEntryPointExtra {
+    function getUserOpHash(PackedUserOperation calldata userOp) external view returns (bytes32);
```

Comment:
what about adding a fuzz test for `getUserOpHash` across versions using known vectors?

Reply:
A fuzz test assumes we have an independant way of checking the produced hash is valid. How would you do that ?